extensions/ql-vscode/src/queries-panel/queries-panel.ts

```diff
@@ -14,4 +20,19 @@ export class QueriesPanel extends DisposableObject {
     });
     this.push(treeView);
   }
+
+  public getCommands(): QueriesPanelCommands {
+    return {
+      "codeQLQueries.runLocalQueryContextInline": this.runLocalQuery.bind(this),
+    };
+  }
+
+  private async runLocalQuery(
+    queryTreeViewItem: QueryTreeViewItem,
+  ): Promise<void> {
+    await this.app.queryServerCommands.execute(
+      "codeQLQueries.runLocalQueryFromQueriesPanel",
+      vscode.Uri.parse(queryTreeViewItem.path),
```

Comment:
It would be safer and more correct to use `Uri.file` here:
```suggestion
      vscode.Uri.file(queryTreeViewItem.path),
```

Reply:
Good point! The "Run query" button currently fails for me (on Windows) with:

![image](https://github.com/github/vscode-codeql/assets/42641846/a48f6834-0abf-450d-8adf-1f7d3528d1eb)


Just tried @robertbrignull's suggestion though, and can confirm that it works 🎉

Reply:
👍